cinema/serializers.py

```diff
@@ -51,7 +59,21 @@
 
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image"
```

Comment:
Ensure that the `image` field in the `MovieDetailSerializer` is not accessible via POST requests to `api/cinema/movies/`, as per the task requirements. This field should only be used via the `/upload-image/` endpoint.